share/vss/vss.go

```diff
@@ -0,0 +1,579 @@
+// vss.go implements the verifiable secret sharing scheme by Pedersen recalled
+// in "Provably Secure Distributed Schnorr Signatures and a (t, n) Threshold
+// Scheme for Implicit Certificates"  by Stinson and Strobl.
+// The basic scheme is taken from Pedersen's paper: "Non-interactive and
+// information-theoretic secure verifiable secret sharing", in Crypto ’91, pages
+// 129–140, 1991.
+// At a higher level, you have one party called the Dealer who wishes to share a
```

Comment:
doc: this will continue the previous paragraph. You might want to add an additional newline here.